src/service/data_products/taxa_count.py

```diff
@@ -208,7 +226,12 @@ async def get_taxa_counts(
             + "Note that if a selection ID is set, any load version override is ignored."),
     status_only: QUERY_VALIDATOR_STATUS_ONLY = False,
     load_ver_override: QUERY_VALIDATOR_LOAD_VERSION_OVERRIDE = None,
-    user: kb_auth.KBaseUser = Depends(_OPT_AUTH)
+    user: kb_auth.KBaseUser = Depends(_OPT_AUTH),
+    sort_priority: list[str] = Query(
+        default=list(_SORT_PRIORITY_ORDER_MAP.keys()),
```

Comment:
The way that fastapi handles lists as query parameters, IIRC, is that for every list entry you have to put another k/v pair into the query like `?sort_priority=match&sort_priority=sel`. The way I've done it in other places is just make it a string and split on commas

Reply:
👍

Reply:
:+1: